packages/core/src/config.ts

```diff
@@ -132,12 +132,27 @@ export function resolveConfig<Theme extends {} = {}>(
   if (!separators.length)
     separators = [':', '-']
 
+  const content = (config.content || {}) as ResolvedConfig['content']
+  content.plain ||= []
+  const defaultPipeline = {
+    include: defaultInclude,
+    exclude: defaultExclude,
+  }
+  content.pipeline ||= defaultPipeline
+  content.pipeline.include ||= defaultPipeline.include
+  content.pipeline.exclude ||= defaultPipeline.include
+
+  // @ts-expect-error private
+  if (config.__postcss)
+    content.filesystem ||= defaultIncludeGlobs
```

Comment:
Could this be outside of the core? Or maybe the entire `content` config could be outside and up to the integrations to resolve

Reply:
Yes, I will move it to the PostCSS plugin.